iceoryx_binding_c/cmake/Config.cmake.in

```diff
@@ -0,0 +1,9 @@
+@PACKAGE_INIT@
```

Comment:
Missing Apache 2 header

Reply:
@budrus @dkroenke  @mossmaurice  @elBoberido  this is a cmake file do we need an Apache Header here too?

If so the cmake files in iceoryx_posh, iceoryx_utils, iceoryx_examples etc. are missing also apache 2.0 header!!

Reply:
Yes cmake files need an Apache header too.